rootfs/templates/wrapper

```diff
@@ -6,6 +6,9 @@ set -o pipefail
 # Geodesic Settings
 export GEODESIC_PORT=${GEODESIC_PORT:-$((30000 + $$ % 30000))}
 
+# If this env var is unset, set it to the current directory without the home dir
+export GEODESIC_WORKDIR=${GEODESIC_WORKDIR:-/localhost/${PWD#$HOME/}}
```

Comment:
I think this default will not make sense if `$PWD` is outside of `$HOME`. That is why in #754 the workdir is set conditionally.

Reply:
@nitrocode

Reply:
The worst thing about this is if you started in a non home directory, you'd simply be placed in `/localhost` which seems acceptable.